index.php

```diff
@@ -838,7 +838,11 @@ private function recursiveDelete($folder) {
 			if ($fileInfo->isDir()) {
 				$directories[] = $fileInfo->getRealPath();
 			} else {
-				$files[] = $fileInfo->getRealPath();
+                if ($fileInfo->isLink()) {
+                    $files[] = $fileInfo->getPathName();
+                } else {
+                    $files[] = $fileInfo->getRealPath();
+                }
```

Comment:
Could I ask you to fix the indentation? We use typically tabs (with a size of 4 spaces per tab).